integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -39,7 +39,7 @@ export function importAppBefore(config: AppConfig | { config: AppConfig }, sdkCo
       throw new Error("Unexpected app on context, use only one importAppBefore per test");
     } else {
       const { appId, baseUrl } = await importApp(config);
-      this.app = new Realm.App({ id: appId, baseUrl, ...sdkConfig });
+      this.app = new Realm.App({ id: appId, baseUrl, baseFilePath: config?.baseFilePath, ...sdkConfig });
```

Comment:
There is a lint error here, but I'm unsure of where it comes from.

```
JsPlatformHelpers.setDefaultRealmFileDirectory is not a function
```

Reply:
Also, I don't think we should be setting `baseFilePath` on `config`, but rather on `sdkConfig`.  I can see how the intention of these parameters are misleading. The `config` is for the `app-importer` to setup the local BaaS. 
 The `sdkConfig` contains the properties for `Realm.App`.  With that being said we should just spread the `sdkConfig` into the `Realm.App`.
 ```suggestion
      this.app = new Realm.App({ id: appId, baseUrl, ...sdkConfig });
```

Reply:
I have reworked in the spirit of your comment